apps/ledger-live-desktop/src/newArch/features/MemoTag/__tests__/MemoTagInfoBody.test.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from "react";
+import { fireEvent, render, screen } from "tests/testUtils";
+import MemoTagInfoBody from "../components/MemoTagInfoBody";
+import { MEMO_TAG_LEARN_MORE_LINK } from "../constants";
+import * as LinkingHelpers from "~/renderer/linking";
+
+jest.mock("~/renderer/linking", () => ({
+  openURL: jest.fn(),
+}));
+
+describe("MemoTagInfoBody", () => {
+  it("should display MemoTagInfoBody correctly", async () => {
+    render(<MemoTagInfoBody />);
+
+    const memoTagInfoBody = screen.getByTestId("memo-tag-info-body");
+    const learnMoreLink = screen.getByText(/Learn more about Tag\/Memo/);
+
+    expect(memoTagInfoBody).toBeVisible();
+    expect(learnMoreLink).toBeVisible();
+
+    fireEvent.click(learnMoreLink);
```

Comment:
for this you can do {user}= render(<MemoTagInfoBody />); at the init of the test
then user.click

Reply:
I realy prefer to rely on native jest/rtl  capability for unit testing component behaviour